application.example.yaml

```diff
@@ -51,6 +51,14 @@ notifications:
     from: x@x.x
     # The email address to send mail TO
     to: x@x.x
+  # Receive instant messages via Telegram whenever there is a trade
+  telegram:
+    # Set to true to enable telegram notifications
+    active: false
+    # Your chat group numeric id with your Telegram bot. Check the wiki page for more information
```

Comment:
I will add it. I thought of only adding the wiki entry when we are about to merge this branch otherwise I am afraid someone may see it in the wiki and try to configure it before this branch is merged. Even though I think very few people check the wiki page :sweat_smile: